### PR TITLE
CMake/MSVC: cleanups around FFMpeg

### PR DESCRIPTION
**Pull request by ligfx**

### CODE REVIEW

**Comment by ligfx** [comment]
@Orphis

**Comment by Orphis** [comment]
https://github.com/hrydgard/ppsspp/blob/master/cmake/Modules/FindFFmpeg.cmake ?

**Comment by ligfx** [comment]
@Orphis I considered moving to a more standard find-module, but this one also handles the pre-built binaries on Windows. We'd have to move that logic somewhere else. I'm leaning towards leaving it as-is, what do you think?

**Comment by Orphis** [comment]
Well, mine handles prebuilt binaries just fine. You can set FFMPEG_DIR variable to the prebuilt location you want, otherwise it will search usual locations.

**Comment by ligfx** [comment]
I didn't realize that you wrote that one! 😆 FFMPEG_DIR works just dandy, updated this PR to use your find-module.

EDIT: Also didn't mean to close it...

File: CMake/FindFFmpeg.cmake [hunk 1]
```diff
@@ -95,7 +95,7 @@ function(find_ffmpeg LIBNAME)
   if(NOT FFMPEG_DIR AND (NOT FFmpeg_LIBRARY_${LIBNAME} OR NOT FFmpeg_INCLUDE_${LIBNAME}))
     # Didn't find it in the usual paths, try pkg-config
     find_package(PkgConfig QUIET)
-    pkg_check_modules(FFmpeg_PKGCONFIG_${LIBNAME} REQUIRED QUIET lib${LIBNAME})
```
Comment (Orphis) [inline CMake/FindFFmpeg.cmake]: Ooops!

Comment (ligfx) [inline CMake/FindFFmpeg.cmake]: :)

**Pull request merged**